src/lib.rs.in

```diff
@@ -140,7 +155,7 @@ impl Handler for WelcomeHandler {
     fn handle(&self, _: &mut Request) -> IronResult<Response> {
         json_response(&ObjectBuilder::new()
             .insert("ladaemon", "Welcome")
-            .insert("version", &self.app.version)
+            .insert("version", env!("CARGO_PKG_VERSION"))
```

Comment:
I figured if it's available at compile time, might as well use that, but we can pass version strings around instead.